Lib/tkinter/test/test_tkinter/test_misc.py

```diff
@@ -48,6 +48,39 @@ def test_tk_setPalette(self):
                 '^must specify a background color$',
                 root.tk_setPalette, highlightColor='blue')
 
+    def test_after_cancel(self):
+        root = self.root
+        timer1 = root.after(5000, lambda: 'break')
+        idle1 = root.after_idle(lambda: 'break')
+
+        # No value for id raises a ValueError.
+        with self.assertRaises(ValueError):
+            root.after_cancel(None)
+
+        # A non-existent id raises a TclError, which is caught in after_cancel.
```

Comment:
The comment looks misleading. This is the test for `after_cancel()` which doesn't raise a TclError.

Actually I think that checking that `root.tk.call('after', 'info', 'spam')` raises a TclError is not needed. Using this command in `after_cancel()` is an implementation detail.

Reply:
Thanks!